src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/CommonResources.resx

```diff
@@ -126,6 +126,9 @@
   <data name="NoBreakingChangesFound" xml:space="preserve">
     <value>APICompat ran successfully without finding any breaking changes.</value>
   </data>
+  <data name="ObsoleteBaselineSuppressionsFound" xml:space="preserve">
+    <value>Obsolete suppressions found that should be removed from suppression files:</value>
```

Comment:
The term `obsolete` doesn't seem correct for these cases.  A term like `unnecessary` might be more clear.  That's the term which is used in https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0079.